sources/Interop/Windows/shared/winerror/Windows.Manual.cs

```diff
@@ -963,361 +961,12902 @@ public static partial class Windows
+
+        public const int DV_E_DVASPECT = unchecked((int)0x8004006B);
+
+        public const int DV_E_DVTARGETDEVICE_SIZE = unchecked((int)0x8004006C);
+
+        public const int DV_E_NOIVIEWOBJECT = unchecked((int)0x8004006D);
+
+        public const long DRAGDROP_E_FIRST = 0x80040100L;
+
+        public const long DRAGDROP_E_LAST = 0x8004010FL;
+
+        public const int DRAGDROP_S_FIRST = 0x00040100;
+
+        public const int DRAGDROP_S_LAST = 0x0004010F;
+
+        public const int DRAGDROP_E_NOTREGISTERED = unchecked((int)0x80040100);
+
+        public const int DRAGDROP_E_ALREADYREGISTERED = unchecked((int)0x80040101);
+
+        public const int DRAGDROP_E_INVALIDHWND = unchecked((int)0x80040102);
+
+        public const int DRAGDROP_E_CONCURRENT_DRAG_ATTEMPTED = unchecked((int)0x80040103);
+
+        public const long CLASSFACTORY_E_FIRST = 0x80040110L;
+
+        public const long CLASSFACTORY_E_LAST = 0x8004011FL;
+
+        public const int CLASSFACTORY_S_FIRST = 0x00040110;
+
+        public const int CLASSFACTORY_S_LAST = 0x0004011F;
+
+        public const int CLASS_E_NOAGGREGATION = unchecked((int)0x80040110);
+
+        public const int CLASS_E_CLASSNOTAVAILABLE = unchecked((int)0x80040111);
+
+        public const int CLASS_E_NOTLICENSED = unchecked((int)0x80040112);
+
+        public const long MARSHAL_E_FIRST = 0x80040120L;
+
+        public const long MARSHAL_E_LAST = 0x8004012FL;
+
+        public const int MARSHAL_S_FIRST = 0x00040120;
+
+        public const int MARSHAL_S_LAST = 0x0004012F;
+
+        public const long DATA_E_FIRST = 0x80040130L;
+
+        public const long DATA_E_LAST = 0x8004013FL;
+
+        public const int DATA_S_FIRST = 0x00040130;
+
+        public const int DATA_S_LAST = 0x0004013F;
+
+        public const long VIEW_E_FIRST = 0x80040140L;
+
+        public const long VIEW_E_LAST = 0x8004014FL;
+
+        public const int VIEW_S_FIRST = 0x00040140;
+
+        public const int VIEW_S_LAST = 0x0004014F;
+
+        public const int VIEW_E_DRAW = unchecked((int)0x80040140);
+
+        public const long REGDB_E_FIRST = 0x80040150L;
+
+        public const long REGDB_E_LAST = 0x8004015FL;
+
+        public const int REGDB_S_FIRST = 0x00040150;
+
+        public const int REGDB_S_LAST = 0x0004015F;
+
+        public const int REGDB_E_READREGDB = unchecked((int)0x80040150);
+
+        public const int REGDB_E_WRITEREGDB = unchecked((int)0x80040151);
+
+        public const int REGDB_E_KEYMISSING = unchecked((int)0x80040152);
+
+        public const int REGDB_E_INVALIDVALUE = unchecked((int)0x80040153);
+
+        public const int REGDB_E_CLASSNOTREG = unchecked((int)0x80040154);
+
+        public const int REGDB_E_IIDNOTREG = unchecked((int)0x80040155);
+
+        public const int REGDB_E_BADTHREADINGMODEL = unchecked((int)0x80040156);
+
+        public const int REGDB_E_PACKAGEPOLICYVIOLATION = unchecked((int)0x80040157);
+
+        public const long CAT_E_FIRST = 0x80040160L;
```

Comment:
`L` is C `long` which is not C# `long`. C `long` is `int` on Windows and `nint` on Unix.

Reply:
`LL` is C `long long` which is C# `long`